### PR TITLE
(bpo-44221) fix ImportError in same situations

### PR DESCRIPTION
**Pull request by luizoti**
https://bugs.python.org/issue44221

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@luizoti

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/asyncio/events.py [hunk 1]
```diff
@@ -118,24 +119,20 @@ def __hash__(self):
...
-        if isinstance(other, TimerHandle):
-            return self._when < other._when
-        return NotImplemented
+        return self._when < other._when
 
     def __le__(self, other):
-        if isinstance(other, TimerHandle):
-            return self._when < other._when or self.__eq__(other)
-        return NotImplemented
+        if self._when < other._when:
```
Comment (eamanu) [inline Lib/asyncio/events.py:125]: We're sure that always other is TimeHandle?

File: Lib/asyncio/events.py [hunk 2]
```diff
@@ -396,7 +393,7 @@ async def create_unix_server(
         The return value is a Server object, which can be used to stop
         the service.
 
-        path is a str, representing a file system path to bind the
+        path is a str, representing a file systsem path to bind the
```
Comment (eamanu) [inline Lib/asyncio/events.py:396]: typo

File: Lib/asyncio/events.py [hunk 3]
```diff
@@ -649,7 +632,7 @@ def get_event_loop(self):
         """
         if (self._local._loop is None and
                 not self._local._set_called and
-                threading.current_thread() is threading.main_thread()):
+                isinstance(threading.current_thread(), threading._MainThread)):
```
Comment (eamanu) [inline Lib/asyncio/events.py:635]: I dind't check deeply but, the intention of the original code is check fi current_thread() is the same object of main_thread(). Why are you change it?

**Review by eamanu — CHANGES_REQUESTED** [review]
Please sign CLA and write the pr title in the form `bpo.NNNN: ....`

**Comment by luizoti** [comment]
@eamanu, now it's correct?

**Comment by domdfcoding** [comment]
I don't see how most of the changes relate to the original issue?

**Comment by eamanu** [comment]
@domdfcoding you're right, This PR/issue it's a little  confuse (for me)

**Comment by luizoti** [comment]
I hadn't noticed the 'hypotheses', the truth is, 'in some situations'

**Comment by luizoti** [comment]
I'm going to test this out during the day, and I'm going to try to give the.  correct follow-up to PR.

**Comment by luizoti** [comment]
I performed more tests, initially this correction took effect, but it seems that it was just a mistake, at this moment the problem continues, I will close this PR and if I can actually find a correction I will submit it again.

**Pull request closed without merging**